### PR TITLE
Spack package for ytopt

### PR DESCRIPTION
**Pull request by pbalapra**
Ytopt package implements search using Random Forest (SuRF), an autotuning search method developed within the ECP ST project Autotuning Compiler Technology for Cross-Architecture Transformation and Code Generation (Ytune).

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-ytopt/package.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#
+#     spack install py-ytopt
+#
+# You can edit this file again by typing:
+#
+#     spack edit py-ytopt
+#
+# See the Spack documentation for more information on packaging.
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/py-ytopt/package.py]: please remove fixme comments :)

Comment (pbalapra) [reply var/spack/repos/builtin/packages/py-ytopt/package.py]: sorry, thanks. I removed them.

**Comment by tgamblin** [comment]
FYI: if you name your branches `packages/*` we'll skip the build part of the travis tests (assuming you only modified/added packages)

**Comment by tgamblin** [comment]
@pbalapra: since you're a new contributor, can you add your consent to #9137?  We are in the midst of a relicense and would like to be able to keep this package after we switch.

**Pull request merged**